profile/README.md

```diff
@@ -0,0 +1,53 @@
+	 * [opendatahub-operator](https://github.com/opendatahub-io/opendatahub-operator): A repository that helps to deploy, monitor and manage the lifecycle of Kubeflow. Built using the Operator Framework which offers an open source toolkit to build, test, package operators and manage the lifecycle of operators.
+	 * [odh-dashboard](https://github.com/opendatahub-io/odh-dashboard): A dashboard for Open Data Hub components. Shows what's installed and what's available for installation, as well as, links to component UIs and links to component documentation.
+	 * [kfp-tekton](https://github.com/opendatahub-io/kfp-tekton): Project bringing Kubeflow Pipelines and Tekton together. The current code allows you run Kubeflow Pipelines with Tekton backend end to end.
+
+  - **Tools**
+  	 *  [kubeflow](https://github.com/opendatahub-io/kubeflow): This repository contains the code of kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
+  	 *  [manifests](https://github.com/opendatahub-io/manifests): The Kubeflow Manifests repository is organized under three (3) main directories, which include manifests for installing: `apps`,  `common`, `contrib`
+  	 *   [odh-images](https://github.com/opendatahub-io/odh-images):  This repository contains code for the various Open Data Hub images.
+  	 * [traefik-proxy](https://github.com/opendatahub-io/traefik-proxy):  This project contains what JupyterHub need to dynamically configure the routes of a traefik proxy server
+
+
+  - **Utilities** 
+	  * [jupyterhub-singleuser-profiles](https://github.com/opendatahub-io/jupyterhub-singleuser-profiles): This library helps to manage and configure singleuser JupyterHub servers deployed by KubeSpawner.
+	  * [s2i-lab-elyra](https://github.com/opendatahub-io/s2i-lab-elyra): This image contains Elyra and all the dependencies and configurations needed to run as a part of OpenDataHub's JupyterHub Environment.
+ 	 * [opendatahub.io](https://github.com/opendatahub-io/opendatahub.io):  Contains the webpage source code 
+ 	 * [opendatahub-community](https://github.com/opendatahub-io/opendatahub-community): This is the starting point for joining and contributing to the Open Data Hub community.
+ 	 * [jupyterhub-odh](https://github.com/opendatahub-io/jupyterhub-odh): This repository contains an example of a customised deployment of JupyterHub for OpenShift, which uses the same OpenShift cluster as the deployment is running in, as the authentication provider.
+
+ - **Template Repositories**
+	 * [odh-template-sig](https://github.com/opendatahub-io/odh-template-sig) Repository template for Open Data Hub Special Interest Groups.
+- **ODH Tutorials**
+	* [Fraud-Detection-with-Business-Workflows-Tutorial](https://github.com/opendatahub-io/Fraud-Detection-with-Business-Workflows-Tutorial)
+	* [data-engineering-and-machine-learning-workshop](https://github.com/opendatahub-io/data-engineering-and-machine-learning-workshop)
+ 	 
```

Comment:
You can delete these because they are no longer current and need to be archived

* Fraud-Detection-with-Business-Workflows-Tutorial
* data-engineering-and-machine-learning-workshop